drivers/SmartThings/matter-pump/profiles/pump-only.yml

```diff
@@ -0,0 +1,19 @@
+name: pump-only
+components:
+- id: main
+  capabilities:
+  - id: switch
+    version: 1
+  - id: pumpOperationMode
+    version: 1
+  - id: pumpControlMode
+    version: 1
+  - id: firmwareUpdate
+    version: 1
+  - id: refresh
+    version: 1
+  categories:
+  - name: Irrigation
+metadata:
+  mnmn: SmartThingsEdge
+  vid: generic-pump-level
```

Comment:
If a custom vid is needed, is this the core one? This profile doesn't have level

Reply:
The reason why the same device configuration is applied to all profiles is that the presentation id of the card is changed when a profile which device configuration is applied is changed from a profile which device configuration is not applied in the driver. and There is no way to check if the presentation id has been changed in the app, so we applied device configuration to all profiles. this is the way proposed by the presentation team, so please let me know if there is a better way.

Reply:
I don't think I was aware of this issue. Let's talk about this internally to see if there are other options

Reply:
@HunsupJung does this mean that both profiles will use the `generic-pump-level` metadata, but the level will still not be displayed for the `pump-only` profile?

Reply:
Yes, that is correct.

Reply:
If we move forward with the custom metadata, will be we able to set things like `capabilities.switchLevel.levelRange()` in the driver, or will the metadata override that? This is similar to the question I asked here: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1458#discussion_r1653382008

Reply:
I have tested this and the answer is yes, we will still be able to use things like the `capabilities.switchLevel.levelRange()` as needed